app/controllers/csv_exports_controller.rb

```diff
@@ -32,7 +32,7 @@ def new
         case params[:type]
         when "users"
           options = user_filter
-          send_data UserCsvPresenter.to_csv(options), type: :csv, filename: "Users_#{options[:datetime]}.csv"
+          send_data UserCsvPresenter.to_csv(**options), type: :csv, filename: "Users_#{options[:datetime]}.csv"
```

Comment:
The `UserCsvPresenter.to_csv` method accepts keyword arguments. Update this method call to match.

https://github.com/zendesk/samson/blob/a370be781f1b0753ea2a0939b6b688fe63eed749/app/presenters/user_csv_presenter.rb#L15-L17